File: tests/fixtures/e2e/java/bad_signature.java
package com.example;

import io.temporal.workflow.WorkflowMethod;
import io.temporal.activity.ActivityMethod;

public interface OrderWorkflow {
    @WorkflowMethod
    String process(String orderId, int quantity, double price);
}

public interface OrderActivities {
    @ActivityMethod
    boolean sendEmail(String to, String subject, String body);
}

File: tests/fixtures/e2e/python/bad_signature.py
from temporalio import workflow, activity


@workflow.defn
class MultiParamWorkflow:
    @workflow.run
    async def run(self, name: str, age: int, active: bool) -> str:
        return f"{name} is {age}"


@activity.defn
async def send_email(to: str, subject: str, body: str) -> bool:
    return True
